src/talk2powersystemllm/app/dev-docker-compose.yaml

```diff
@@ -0,0 +1,36 @@
+services:
+
+  redis:
+      image: redis:8.0.3-alpine
+      container_name: redis
+      command: >
+        redis-server --requirepass DUMMY_REDIS_PASSWORD --appendonly yes --appendfsync everysec --save 300 10 --save 60 10000
+      ports:
+        - "6379:6379"
+      volumes:
+        - redis_data:/data
+
+  talk2powersystem:
+    image: talk2powersystem
+    container_name: talk2powersystem
+    environment:
+      - AGENT_CONFIG=/code/config/cim.ontotext.no.cognite.yaml
+      - GRAPHDB_PASSWORD=<GRAPHDB_PASSWORD_FROM_KEEPER>
```

Comment:
Could be extended to read these from a `.env` file and avoid updating/replacing this file.

Reply:
Done